src/ralph/models/conversion_set.py

```diff
@@ -0,0 +1,56 @@
+"""Base conversion set definition"""
```

Comment:
I would move this module content to `src/ralph/models/converter.py`. It's not that big for now.

Reply:
Good Idea) Thanks) Updated)